inference-engine/tests/functional/plugin/cpu/single_layer_tests/prior_box_clustered.cpp

```diff
@@ -0,0 +1,228 @@
+            attributes.widths,
+            attributes.heights,
+            attributes.clip,
+            attributes.step_widths,
+            attributes.step_heights,
+            attributes.step,
+            attributes.offset,
+            attributes.variances) = specParams;
+
+        std::ostringstream result;
+        const char separator = '_';
+
+        result << "IS="      << inputShapes << separator;
+        result << "imageS="  << imageShapes << separator;
+        result << "netPRC="  << netPrecision << separator;
+        result << "inPRC="   << inPrc << separator;
+        result << "outPRC="  << outPrc << separator;
+        result << "inL="     << inLayout << separator;
+        result << "outL="    << outLayout << separator;
+        result << "widths="  << CommonTestUtils::vec2str(attributes.widths)  << separator;
+        result << "heights=" << CommonTestUtils::vec2str(attributes.heights) << separator;
+        result << "variances=";
+        if (attributes.variances.empty())
+            result << "()" << separator;
+        else
+            result << CommonTestUtils::vec2str(attributes.variances) << separator;
+        result << "stepWidth="  << attributes.step_widths  << separator;
+        result << "stepHeight=" << attributes.step_heights << separator;
+        result << "step="       << attributes.step << separator;
+        result << "offset="     << attributes.offset << separator;
+        result << "clip="       << std::boolalpha << attributes.clip << separator;
+        result << "trgDev="     << targetDevice;
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        priorBoxClusteredSpecificParams specParams;
+
+        InferenceEngine::Layout inLayout;
+        InferenceEngine::Layout outLayout;
+        ov::test::ElementType netPrecision;
+        ov::test::ElementType inPrc;
+        ov::test::ElementType outPrc;
+        ov::test::InputShape inputShapes;
+        ov::test::InputShape imageShapes;
+        std::tie(specParams, netPrecision,
+                 inPrc, outPrc, inLayout, outLayout,
+                 inputShapes, imageShapes, targetDevice) = GetParam();
+
+        selectedType = makeSelectedTypeStr("ref", inPrc);
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+
+        init_input_shapes({ inputShapes, imageShapes });
+
+        ngraph::op::PriorBoxClusteredAttrs attributes;
+        std::tie(
+            attributes.widths,
+            attributes.heights,
+            attributes.clip,
+            attributes.step_widths,
+            attributes.step_heights,
+            attributes.step,
+            attributes.offset,
+            attributes.variances) = specParams;
+
+        auto params = ngraph::builder::makeDynamicParams(netPrecision, { inputShapes.first, imageShapes.first });
+
+        auto shape_of_1 = std::make_shared<ngraph::opset3::ShapeOf>(params[0]);
+        auto shape_of_2 = std::make_shared<ngraph::opset3::ShapeOf>(params[1]);
```

Comment:
do we really need to make subgraph with `shape of`?

Reply:
please, look at the next comment